platform/client-admin/src/forms/createVolunteerAction.ts

```diff
@@ -0,0 +1,44 @@
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import SimpleSchema from 'simpl-schema';
+
+export default function createVolunteerActionForm(recipients: string[], volunteers: string[], distributionCentres: string[], products: string[]) {
+    const volunteerActionForm = new SimpleSchema({
+        title: {
+            type: String,
+            max: 120
+        },
+        description: {
+            type: String,
+            max: 120
+        },
+        recipientName: {
+            type: String,
+            max: 220,
+            uniforms: {
+                allowedValues: recipients
+            }
+        },
+        volunteerName: {
+            type: String,
+            max: 220,
+            uniforms: {
+                allowedValues: volunteers
+            }
+        },
+        distributionCentreName: {
+            type: String,
+            max: 220,
+            uniforms: {
+                allowedValues: distributionCentres
+            }
+        }/*,
```

Comment:
See https://github.com/aerogear/OpenVolunteerPlatform/commit/ee835b35b7beff4da3ad55ce74cdb03e283c11e9#r39254476

Reply:
So this will remain commented till the upstream issue is fixed, I have also created https://github.com/aerogear/OpenVolunteerPlatform/issues/69#issue-620903015 to gather ideas about how products should be linked to a volunteer action.